### PR TITLE
Change version to 0.0.2

### PR DESCRIPTION
**Pull request by DarkaMaul**
Prepare release 0.0.2.

### CODE REVIEW

File: src/rfc3161_client/__init__.py
Comment (woodruffw) [inline src/rfc3161_client/__init__.py:34]: FWIW I'm OK with just removing this -- if we can't single-source the version, the `__version__` attribute is less important (since it's just a convention).

File: CHANGELOG.md
Comment (facutuesca) [inline CHANGELOG.md:21]: We can probably remove this, since it's not a user-facing change
```suggestion
```

File: CHANGELOG.md
Comment (facutuesca) [inline CHANGELOG.md:12]:
```suggestion
- Magic methods (`__hash__` and `__repr__`) have been added for TimestampResponse and 
  TimestampRequest ([#32](https://github.com/trailofbits/rfc3161-client/pull/32))
```

File: CHANGELOG.md
Comment (facutuesca) [inline CHANGELOG.md:17]:
```suggestion
- The version is now correctly sourced from `pyproject.toml` ([#30](https://github.com/trailofbits/rfc3161-client/pull/30))
- The nonce generation no longer fails sporadically ([#33](https://github.com/trailofbits/rfc3161-client/pull/33))
```

File: CHANGELOG.md
Comment (facutuesca) [inline CHANGELOG.md:22]:
```suggestion
- The public API is now available directly from the main package module ([#36](https://github.com/trailofbits/rfc3161-client/pull/36))
```

File: CHANGELOG.md
Comment (facutuesca) [inline CHANGELOG.md:26]: Missing links for the headers:
```suggestion
This is the first alpha release of `rfc3161-client`.

[Unreleased]: https://github.com/trailofbits/rfc3161-client/compare/v0.0.1...HEAD
[0.0.1]: https://github.com/trailofbits/rfc3161-client/releases/tag/v0.0.1
```

File: CHANGELOG.md
Comment (facutuesca) [inline CHANGELOG.md:7]: If this PR is going to be the last one before releasing 0.0.2, we can change `Unreleased` to `0.0.2`

**Review by facutuesca — COMMENTED** [review]
Thanks for adding the changelog! I left some wording suggestions

**Comment by DarkaMaul** [comment]
I will wait until #35 is merged before merging this one and releasing 0.0.2.

File: CHANGELOG.md
Comment (facutuesca) [inline CHANGELOG.md:34]:
```suggestion
[0.0.2]: https://github.com/trailofbits/rfc3161-client/compare/v0.0.1...v0.0.2
```

**Review by facutuesca — APPROVED** [review]
LGTM!

**Pull request merged**
